app/models/report/summary_collection_builder.rb

```diff
@@ -14,6 +14,9 @@ class Report::SummaryCollectionBuilder
     'long_text' => 'raw'
   }
 
+  # Quantity of raw answers that should be shown on report
+  RAW_ANSWERS_LIMIT = 100
```

Comment:
Perhaps same here: limit to 101? Are we currently showing an indication that some answers were clipped? This is important I think.